apollo-ios-codegen/Sources/ApolloCodegenLib/CodegenConfiguration/ApolloCodegenConfiguration+RenameSchemaTypesConfiguration.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+struct SchemaCustomization: Codable {
```

Comment:
Figured we could name this `SchemaCustomizations` so it could potentially contain more options in the future.